lib/matplotlib/backend_bases.py

```diff
@@ -3428,7 +3422,13 @@ def add_tool(self, name, tool, group=None, position=None):
                           'not added')
             return
 
-        self._tools[name] = tool_cls(self, name)
```

Comment:
I really don't like the idea of opening the door to transfter tools between navigations..... I don't know....